modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js

```diff
@@ -125,18 +125,24 @@ export default function TrafficSources({
 												)
 											}
 										>
-											<ClayButton
-												className="font-weight-semi-bold px-0 py-1 text-primary"
-												displayType="link"
-												onClick={() =>
-													onTrafficSourceClick(
-														entry.name
-													)
-												}
-												small
-											>
-												{entry.title}
-											</ClayButton>
+											{!entry.countryKeywords.length && (
```

Comment:
Maybe a ternary operator would look better here?

```js
{entry.countryKeywords.length ? (
    <Foo />
) : (
    <Bar />
)}

Reply:
In this case, I'll change it to a ternary operator. If the conditions are a little bit tricky and the code in each case is long, I try to not use ternary because I read it better separately.